cmd/janus/init.go

```diff
@@ -27,7 +27,7 @@ func initConfig() {
 	var err error
 	globalConfig, err = config.Load(configFile)
 	if nil != err {
-		log.WithError(err).Error("Could not load configurations from file")
+		log.WithError(err).Error("Could not load configurations from file - trying MongoDB configurations instead.")
```

Comment:
That's not really the behavior that happens here. There is no MongoDB configurations loading... Either is from a file or from Environment Variables

Reply:
Thanks @italolelis. I changed the message a little bit.